CHANGELOG.md

```diff
@@ -32,6 +32,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Changed type signature of `sys::select::FdSet::contains` to make `self`
   immutable ([#564](https://github.com/nix-rust/nix/pull/564))
 - Changed type of `sched::sched_setaffinity`'s `pid` argument to `pid_t`
+- Newtypes for UID, GID, and PID. `uid_t` changed to `Uid`, `gid_t` changed to
```

Comment:
Please link to the pull request like the other changelog entries do. Additionally I'd rephrase this to be: "Introduced wrapper types for `gid_t`, `pid_t`, and `uid_t` as `Gid`, `Pid`, and `Uid` respectively. Various functions have been changed to use these new types as arguments. (PULL_REQUEST_LINK)"